cilium-cli/encrypt/ipsec_rotate_key.go

```diff
@@ -98,9 +90,7 @@ func keyFromSlice(parts []string) (ipsecKey, error) {
 	if len(parts) != 5 {
 		return ipsecKey{}, fmt.Errorf("IPsec key invalid [expected parts: 5, actual parts: %d]", len(parts))
 	}
-	spiSuffix := false
 	if strings.HasSuffix(parts[1], "+") {
```

Comment:
Can't this condition go away as you did in `cipherKeyFromSlice`?

Reply:
Ah, I just saw this is done, but in the wrong commit :slightly_smiling_face:

Reply:
My bad, fixed! Thanks!